channeldb/payments.go

```diff
@@ -460,7 +460,7 @@ type PaymentsQuery struct {
 type PaymentsResponse struct {
 	// Payments is the set of payments returned from the database for the
 	// PaymentsQuery.
-	Payments []MPPayment
+	Payments []*MPPayment
```

Comment:
is there a tangible performance benefit here? the `MPPayment` struct is only ~50 bytes

Reply:
Not a massive savings, but might as well eliminate some low hanging fruit.

Reply:
in general i prefer not to have arrays of pointers, easy to end up with weird behavior